base/settings.jl

```diff
@@ -0,0 +1,11 @@
+baremodule Settings
+using Core: ARGS, include, Intrinsics, memoryrefget, memoryref
+
+MAX_METHODS::Int = 3
+UNOPTIMIZE_THROW_BLOCKS::Bool = true
+INCLUDE_STDLIBS::String = "FileWatching,Libdl,Artifacts,SHA,Sockets,LinearAlgebra,Random"
+
+if Intrinsics.slt_int(1, getfield(getfield(ARGS, :size), 1))
+    include(Settings, memoryrefget(memoryref(ARGS.ref, 2, false), :not_atomic, false))
+end
```

Comment:
What's this magic here?

Reply:
this is an array getindex that is hard-coded presumably due to bootstraping issues.

Reply:
specifically, this is
```
if 1 <= ARGS.size[1]
    include(Settings, ARGS[2])
end
```